docs/attributes-registry/feature-flag.md

```diff
@@ -41,3 +42,19 @@ For example, the variant `red` maybe be used for the value `#c05543`.
 | `static` | The resolved value is static (no dynamic evaluation). | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `targeting_match` | The resolved value was the result of a dynamic evaluation, such as a rule or specific user-targeting. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | `unknown` | The reason for the resolved value could not be determined. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+---
+
+`feature_flag.evaluation.value.type` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
```

Comment:
It's not for this PR, but we're seriously considering allowing complex attribute types **instead of body**

https://github.com/open-telemetry/semantic-conventions/issues/1669 https://github.com/open-telemetry/opentelemetry-specification/pull/4334.

Wonder what your thoughts are